common/components/componentsBySection/FindGroups/GroupCard.jsx

```diff
@@ -95,18 +94,28 @@ class GroupCard extends React.PureComponent<Props> {
     let issueNames: $ReadOnlyArray<string> = Sort.byCountDictionary(
       this.props.group.group_issue_areas
     );
-    issueNames = issueNames.map(
-      (issueName: string) => this.props.tagDictionary[issueName].display_name
-    );
-    issueNames = Truncate.arrayT(issueNames, this.props.maxIssuesCount);
 
-    return (
-      <ul>
-        {issueNames.map((issueName: string, i: number) => (
-          <li key={i}>{issueName}</li>
-        ))}
-      </ul>
-    );
+    if (!_.isEmpty(issueNames)) {
+
+      issueNames = issueNames.map(
+        (issueName: string) => this.props.tagDictionary[issueName].display_name
+      );
+      issueNames = Truncate.arrayT(issueNames, this.props.maxIssuesCount);
+
+      return (
+        <div>
```

Comment:
```suggestion
        <React.Fragment>
```
No need for an extra div just for enveloping the content, you can just use React.Fragment